### PR TITLE
Add ente-auth extension

### PR DESCRIPTION
**Pull request by chkpwd**
## Description

Ente Auth TOTP generator to display TOTP codes. This does not use any API exposed from the upstream project. You have to import your secrets using the import subcommand built into the extension. See screenshots below.

## Screencast
ente-auth-1
ente-auth-2
ente-auth-3
ente-auth-4
ente-auth-5

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by talalong** [comment]
This could be the replacement for my Authy extension. Thank you for your work. Cant wait to test it.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Could you look into handling errors gracefully 🙂

ente-auth 2024-10-11 at 11 42 53

**Comment by S33G** [comment]
Love this.
Code looks good to me

File: extensions/ente-auth/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Ente Auth Changelog
+
```
Comment (S33G) [inline extensions/ente-auth/jest.config.js:2]: This doesn't seem right? :D

File: extensions/ente-auth/src/helper/ente.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  }
+
+  return path;
+};
+
+export const checkEnteBinary = (): void => {
+  try {
+    execSync("ente version");
+  } catch (error) {
+    console.log("Ente binary not found. Please install it.");
```
Comment (S33G) [inline extensions/ente-auth/src/helper/ente.ts:20]: This check just console logs and needs to handle the error :)

**Comment by chkpwd** [comment]
@pernielsentikaer I appreciate the feedback. Any else I should change besides error handling?

**Comment by pernielsentikaer** [comment]
I don't think so but @S33G mentions a few things too

**Comment by pernielsentikaer** [comment]
@S33G do you mind checking again 🙂

**Comment by calebjonasson** [comment]
The updated changes look good to me! 🚀

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**